libs/designsystem/src/lib/components/list/helpers/list-helper.ts

```diff
@@ -1,5 +1,5 @@
-import { ListComponent } from '../list.component';
 import { LoadOnDemandEventData } from '../list.event';
+import { ListComponent } from '../list.component';
```

Comment:
Seems like the order has been reversed? Please revert :)

Reply:
It's intentional - I have reverted the file in commit fdaac63 as no other changes than reordering imports are being applied to that file 😄 

It should be identical to `list-helper.ts` on master now.